### PR TITLE
out_rdkafka2: Follow rdkafka's log level for rdkafka log

### PR DESCRIPTION
**Pull request by repeatedly**
Signed-off-by: Masahiro Nakagawa <repeatedly@gmail.com>

### CODE REVIEW

File: lib/fluent/plugin/out_rdkafka.rb [hunk 1]
```diff
@@ -91,8 +92,22 @@ def configure(conf)
     super
     log.instance_eval {
       def add(level, &block)
-        if block
+        return unless block
+
+        # Follow rdkakfa's log level. See also rdkafka-ruby's bindings.rb.
```
Comment (ganmacs) [inline lib/fluent/plugin/out_rdkafka.rb]: I think it's better to have the permanent link which has commit hash to bindings.rb.

Comment (repeatedly) [reply lib/fluent/plugin/out_rdkafka.rb]: Added github link

File: lib/fluent/plugin/out_rdkafka.rb [hunk 2]
```diff
@@ -91,8 +92,22 @@ def configure(conf)
     super
     log.instance_eval {
       def add(level, &block)
-        if block
+        return unless block
```
Comment (ganmacs) [inline lib/fluent/plugin/out_rdkafka.rb]: [nits] Can we use `block_given?`?

Comment (repeatedly) [reply lib/fluent/plugin/out_rdkafka.rb]: Does it mean change method args like below?

```
def add(level)
  return unless block_given?
```

Comment (ganmacs) [reply lib/fluent/plugin/out_rdkafka.rb]: I thought just changing to `return unless block_given?` part. but yours also is fine for me.

**Pull request merged**